stanford/playbooks/coursegraph.yml

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: coursegraph
+  when: redeploy | default(False) | bool
+- name: "Setup {{ COMMON_HOSTNAME }}"
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: true
+  vars:
+    CLUSTER_NAME: coursegraph
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+  roles:
+    - neo4j
```

Comment:
Since we're not opening this up publicly, I don't think we need the nginx role anymore.
All it did was proxy port 80 -> 7474, which we weren't even using in testing anyway...

Reply:
Since 7474 is http, should we be using 7473 (https)?

Reply:
No, we don't need to set up an SSL cert for this.
Per our planned internal rollout of this, we're only going to be accessing this machine over SSH,
so we'll already have encryption across-the-wire.
We're only using unencrypted/http directly on the localhost, not across the network.